doc/utils/sui-setup.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# Prereqs: Git CLI and GitHub account
+# Usage: set up environment for Sui development
+# Run `sui-setup.sh` in the directory to download source
+shopt -s nullglob
+set -e
+set -o pipefail
+
+## Get Cargo with Rust toolchain
+curl https://sh.rustup.rs -sSf | sh
```

Comment:
We should probably check if the user already has Rust installed before running this command.

I have Rust already installed, and when I run this I got

```
➜ curl https://sh.rustup.rs -sSf | sh
info: downloading installer

Welcome to Rust!

This will download and install the official compiler for the Rust
programming language, and its package manager, Cargo.

Rustup metadata and toolchains will be installed into the Rustup
home directory, located at:

  /Users/cl/.rustup

This can be modified with the RUSTUP_HOME environment variable.

The Cargo home directory located at:

  /Users/cl/.cargo

This can be modified with the CARGO_HOME environment variable.

The cargo, rustc, rustup and other commands will be added to
Cargo's bin directory, located at:

  /Users/cl/.cargo/bin

This path will then be added to your PATH environment variable by
modifying the profile files located at:

  /Users/cl/.profile
  /Users/cl/.zshenv

You can uninstall at any time with rustup self uninstall and
these changes will be reverted.

Current installation options:


   default host triple: aarch64-apple-darwin
     default toolchain: stable (default)
               profile: default
  modify PATH variable: yes

```

which requires human input and seems to try to install again

Reply:
Thanks, Chris. I've fixed this by instead simply checking for install and if not asking them to do it manually.